### PR TITLE
Add UI intro/tour

### PR DESCRIPTION
**Pull request by remingtonc**
Uses intro.js to do a tour of features.

### CODE REVIEW

File: web/src/web/templates/base.html [hunk 1]
```diff
@@ -14,48 +15,48 @@
...
                     </li>
-                    <li class="sidebar__item">
+                    <li class="sidebar__item" data-step="2" data-position="right" data-intro="Use Search to find XPaths, OIDs, whatever you are looking for more easily with keywords!">
                         <a href="{{ url_for('search_es') }}">
                             <span class="icon-search"></span>
                             <span class="half-margin-left">Search</span>
                         </a>
                     </li>
-                    <li class="sidebar__item">
+                    <li class="sidebar__item" data-step="3" data-position="right" data-intro="Matchmaker helps take a list of datapaths like OIDs or XPaths and find what's been mapped as equivalent. Great for upgrading a list of OIDs to XPaths.">
```
Comment (ziausyed) [inline web/src/web/templates/base.html]: Did you mean converting or is upgrading the right word to use here?

File: web/src/web/templates/base.html [hunk 2]
```diff
@@ -14,48 +15,48 @@
...
                         </a>
-                    <li class="sidebar__item">
+                    <li class="sidebar__item" data-step="7" data-position="right" data-intro="Learn more about TDM!">
                         <a href="/doc/" target="_blank">
                             <span class="icon-lightbulb"></span>
                             <span class="half-margin-left">Documentation</span>
                         </a>
                     </li>
-                    <li class="sidebar__item">
+                    <li class="sidebar__item" data-step="8" data-position="right" data-intro="Backup your mappings, or restore/merge mappings from a previous backup.">
```
Comment (ziausyed) [inline web/src/web/templates/base.html]: Is there a way for the "next" button on this one to not show up since it's the last option?

Comment (remingtonc) [reply web/src/web/templates/base.html]: Good idea, @apletcher had the same feedback. There's some funky CSS magic happening which is making this difficult.

File: web/src/web/templates/base.html (hunk 1 above)
Comment (remingtonc) [reply web/src/web/templates/base.html]: If we're speaking technically, convert is the right word, but the marketer in me likes upgrade.

File: web/src/web/templates/base.html (hunk 2 above)
Comment (remingtonc) [reply web/src/web/templates/base.html]: Resolved.

**Pull request merged**